apps/communications/contacts/test/unit/mock_activities.js

```diff
@@ -5,6 +5,7 @@
 var MockActivityHandler = {
   currentlyHandling: false,
   activityName: 'view',
+  activityDataType: 'text/vcard',
```

Comment:
better to do this inside a specific test instead of modifying the whole mock which can lead to unintended side effects